src/frontend/planner_test/tests/testdata/join.yaml

```diff
@@ -238,16 +232,16 @@
             BatchExchange { order: [], dist: HashShard(i.x) }
               BatchScan { table: i, columns: [i.x], distribution: UpstreamHashShard(i.x) }
   stream_plan: |
-    StreamMaterialize { columns: [x, i.t._row_id(hidden), i.t._row_id#1(hidden), i.t._row_id#2(hidden), i.t._row_id#3(hidden)], pk_columns: [i.t._row_id, i.t._row_id#1, i.t._row_id#2, i.t._row_id#3] }
-      StreamExchange { dist: HashShard(i.t._row_id, i.t._row_id, i.t._row_id, i.t._row_id) }
-        StreamProject { exprs: [Coalesce(i.x, i.x), i.t._row_id, i.t._row_id, i.t._row_id, i.t._row_id] }
-          StreamHashJoin { type: FullOuter, predicate: i.x = i.x, output: [i.x, i.x, i.t._row_id, i.t._row_id, i.t._row_id, i.t._row_id] }
-            StreamHashJoin { type: Inner, predicate: i.x = i.x, output: [i.x, i.t._row_id, i.t._row_id] }
+    StreamMaterialize { columns: [x, i.t._row_id(hidden), i.t._row_id#1(hidden), i.x(hidden), i.x#1(hidden), i.t._row_id#2(hidden), i.t._row_id#3(hidden), i.x#2(hidden), i.x#3(hidden)], pk_columns: [i.t._row_id, i.t._row_id#1, i.x, i.x#1, i.t._row_id#2, i.t._row_id#3, i.x#2, i.x#3] }
+      StreamExchange { dist: HashShard(i.t._row_id, i.t._row_id, i.x, i.x, i.t._row_id, i.t._row_id, i.x, i.x) }
+        StreamProject { exprs: [Coalesce(i.x, i.x), i.t._row_id, i.t._row_id, i.x, i.x, i.t._row_id, i.t._row_id, i.x, i.x] }
+          StreamHashJoin { type: FullOuter, predicate: i.x = i.x, output: [i.x, i.x, i.t._row_id, i.t._row_id, i.x, i.t._row_id, i.t._row_id, i.x] }
+            StreamHashJoin { type: Inner, predicate: i.x = i.x, output: [i.x, i.t._row_id, i.t._row_id, i.x] }
```

Comment:
A lot of `i.x` here. 😄 Do we have an issue for this?

Reply:
Could you please also add a test for a trivial `FullOuter` join? Like 

https://github.com/risingwavelabs/risingwave/blob/7d2651c122bfb11b21f3d4c6eb34ec0625174465/e2e_test/streaming/join.slt#L70-L95

Reply:
maybe I will refine the join's planner test later...